core/Scheduler/Schedule/OneTime.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Scheduler\Schedule;
+
+class OneTime extends Schedule
```

Comment:
I was wondering if it is maybe a bit misleading? As a developer might expect this to be executed only once, but when using something like `$schedule = new OneTime(Date::now()->getTimestamp());` it would be executed more than once?

Reply:
Hmm, could rename it to `FixedTime`?

Reply:
Will rename it to `SpecificTime`, hopefully will be clearer.